### PR TITLE
【Error Message No. 17】Replace part of CHECK_ in paddle/cinn/frontend/decomposer/*

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR types

Others
### PR changes

Others
### Description

https://github.com/PaddlePaddle/Paddle/issues/62748

### CODE REVIEW

**Comment by luotao1** [comment]
@Liyulingyue 张师傅，你做的应该不是任务1吧？

File: paddle/cinn/frontend/decomposer/broadcast.cc [hunk 1]
```diff
@@ -51,10 +52,18 @@ void GetReduceDimsForY(const std::vector<int>& dy_shape,
 
 void elementwise_add(const Instruction& instr,
                      const DecomposerContext& context) {
-  CHECK_EQ(instr->inputs.size(), 2UL)
-      << " 2 input tensors for " << instr->op_type;
-  CHECK_EQ(instr->outputs.size(), 1UL)
-      << "1 output tensor for " << instr->op_type;
+  PADDLE_ENFORCE_EQ(instr->inputs.size(),
+                    2UL,
+                    platform::errors::InvalidArgument(
```
Comment (Liyulingyue) [inline paddle/cinn/frontend/decomposer/broadcast.cc]:
```suggestion
                    phi::errors::InvalidArgument(
```

File: paddle/cinn/frontend/decomposer/broadcast.cc [hunk 2]
```diff
@@ -51,10 +52,18 @@ void GetReduceDimsForY(const std::vector<int>& dy_shape,
...
+                        "The size of inputs in elementwise_add is incorrect. "
+                        "Expected size is 2, but receive %d. ",
+                        instr->inputs.size()));
+  PADDLE_ENFORCE_EQ(instr->outputs.size(),
+                    1UL,
+                    platform::errors::InvalidArgument(
```
Comment (Liyulingyue) [inline paddle/cinn/frontend/decomposer/broadcast.cc]:
```suggestion
                    phi::errors::InvalidArgument(
```

File: paddle/cinn/frontend/decomposer/broadcast.cc [hunk 3]
```diff
@@ -120,10 +129,20 @@ void elementwise_add(const Instruction& instr,
...
 void elementwise_add_grad(const Instruction& instr,
                           const DecomposerContext& context) {
-  CHECK_EQ(instr->inputs.size(), 3UL)
-      << " 3 input tensors for " << instr->op_type;
-  CHECK_EQ(instr->outputs.size(), 2UL)
-      << "2 output tensors for " << instr->op_type;
+  PADDLE_ENFORCE_EQ(
+      instr->inputs.size(),
+      3UL,
+      platform::errors::InvalidArgument(
```
Comment (Liyulingyue) [inline paddle/cinn/frontend/decomposer/broadcast.cc]:
```suggestion
      phi::errors::InvalidArgument(
```

File: paddle/cinn/frontend/decomposer/broadcast.cc [hunk 4]
```diff
@@ -120,10 +129,20 @@ void elementwise_add(const Instruction& instr,
...
+          "The size of inputs in elementwise_add_grad is incorrect. "
+          "Expected size is 3, but receive %d. ",
+          instr->inputs.size()));
+  PADDLE_ENFORCE_EQ(
+      instr->outputs.size(),
+      2UL,
+      platform::errors::InvalidArgument(
```
Comment (Liyulingyue) [inline paddle/cinn/frontend/decomposer/broadcast.cc]:
```suggestion
      phi::errors::InvalidArgument(
```

**Review by Galaxy1458 — APPROVED** [review]

**Pull request merged**